dspace/etc/postgres/README.md

```diff
@@ -31,3 +31,31 @@ feature).
 This `update-sequences.sql` script can be run manually. It will not harm your 
 database (or its contents) in any way. It just ensures all database counts (i.e.
 sequences) are properly set to the next available value.
+
+## Using the ex- and import-c-c.sql script
+The ex- and import-c-c.sql contains SQL scripts for export and import of the tree of communities and collections into a new fresh DSpace installation.
+
+This can preferably be used to create a basic data set for test and development systems, containing the same community-collection-tree as in the production system but without any individual-related data.  
+  
+### Preconditions
+* There exists (on the source as well as on the target server) a directory `/tmp/dspace-export`. On the source server must the user postgres be allowed to write to this directory.
```

Comment:
I like the concept of these scripts overall. But, I wonder if there's a way to make the directory it writes to configurable?  Perhaps even just writing the exported files to the current directory?

Reply:
I have to test this, but I think the current directory is something that should be doable. Before I invest more time into this, I'd like to know if this is going to be merged or not (see comments in the ticket and bellow).